spec/lib/vet360/models/address_spec.rb

```diff
@@ -30,43 +40,43 @@
 
   describe 'validation' do
     context 'for any type of address' do
-      it 'address_pou is requred', :aggregate_failures do
+      it 'address_pou is requred' do
```

Comment:
did you consider moving `:aggregate_failures` to the validation block instead of to the whole test suite?